api-reference/beta/api/subscription_delete.md

```diff
@@ -2,7 +2,22 @@
 
 Delete a subscription.
 ## Prerequisites
-One of the following **scopes**, depending on the target resource, are required to execute this API: *Mail.Read*, *Calendars.Read*, *Contacts.Read*, *Group.Read.All*, *Files.ReadWrite*, or *Files.ReadWrite.All*
+
+One of the following **scopes**, depending on the target resource, are required to execute this API. Creating a subscription requires read scope to the resource. For example, to get notifications messages, your app needs the `Mail.Read` permission. The following table lists the suggested roles needed for each resource.
+
+|Resource type or Item | Scope or role |
+|--------------|--------------------|
+| Messages     | Mail.Read          |
+| Calendars    | Calendars.Read     |
+| Contacts     | Contacts.Read      |
+| Files        | Files.ReadWrite    |
+| Users        | User.Read.All      |
+| Groups       | Group.Read.All     |
+| Drive root   | Files.ReadWrite    |
+| Drives associated with Sharepoint sites | Files.ReadWrite.All |
+
+***Note:*** The /beta endpoint allows Application permissions for most resources. Conversations in a Group and OneDrive drive root items are not supported with Application permissions.
+
```

Comment:
So there are a few things here (please see email thread for details):
1. We probably won't update Prereq sections in other topics for a few weeks. I'm not sure if you want to lead the charge -- but up to you.
2. We don't want to talk about "roles".
3. Laura shut me down big time on capitalizing "Application" and "Delegated" unless it's at the beginning of a sentence.
4. We somehow lost information about which APIs support consumer scenarios in this -- I can guess based on the permission. I don't think our customers can :).
Maybe we should sort this out in email -- first?